lib/bookingsync/engine/credentials_resolver.rb

```diff
@@ -0,0 +1,18 @@
+class BookingSync::Engine::CredentialsResolver
+  attr_accessor :host
```

Comment:
Why not `attr_reader`? Do we ever use credentials_resolver.host = some_host that we need it?

It should be probably private as well